src/api/public/apidocs-new/paths/search_owner.yaml

```diff
@@ -1,32 +1,39 @@
 get:
-  summary: Search for the default responsible person or group (the owner).
+  summary: Search for the users or groups who are the owners of a project or package.
   description: |
-    Search for the default responsible person or group (the owner) by a user, group
-    or binary name.
+    The owners are users/groups with the maintainer or bugowner roles in the given project/package.
+
+    The owner search will get results only if you specify a project or if one of the ancestors or
+    related projects of the searched package/project was previously set as the root of the search.
```

Comment:
```suggestion
   By default the owner search will get results from projects (and their sub-projects) that have the OBS:OwnerRootProject attribute.
```

Reply:
Then add to the rest to the `project` parameter below

```
By default the owner search will get results from projects (and their sub-projects) that have the OBS:OwnerRootProject attribute. Use this parameter to overwrite the default and search in this project (and it's sub-projects).
```

Reply:
done